deploy/helm/lunettes/templates/elasticsearch/elasticsearch-clean-job.yaml

```diff
@@ -1,4 +1,8 @@
+{{- if semverCompare ">=1.21-0" .Capabilities.KubeVersion.GitVersion -}}
 apiVersion: batch/v1
+{{- else if semverCompare ">=1.19-0" .Capabilities.KubeVersion.GitVersion -}}
+apiVersion: batch/v1beta1
```

Comment:
Is there any other diff with "v1beta1" and "v1" of CronJob?

Reply:
From k8s 1.19 to 1.29, there are only v1beta1 and v1